app/views/widgets/onlineTestProgress.scala.html

```diff
@@ -1,11 +1,16 @@
 @import connectors.ExchangeObjects.OnlineTest
+@import security.Roles.FastPassReceivedRole
 @(usr: CachedData, onlineTest: Option[OnlineTest] = None,
         applicationStatus: models.ApplicationData.ApplicationStatus.ApplicationStatus,
         allocationDetails: Option[connectors.AllocationExchangeObjects.AllocationDetails])(implicit request: Request[_])
 
 @import models.ApplicationData.ApplicationStatus._
 
 @applicationStatus match {
+    case _ if FastPassReceivedRole.isAuthorized(usr) => {
```

Comment:
Is there any chance this could be moved to the bottom of the match statement with similar case _ statements and messages? I'm not super attached to it, but it will make the file easier to read IMHO